openmc/material.py

```diff
@@ -422,7 +427,15 @@ def add_element(self, element, percent, percent_type='ao'):
         else:
             element = openmc.Element(element)
 
-        self._elements[element._name] = (element, percent, percent_type)
+        if expand:
+            if percent_type == 'wo':
+                raise NotImplementedError('Expanding natural element based on '
+                                          'weight percent is not yet supported.')
+            for isotope, abundance in element.expand():
+                self._nuclides[isotope.name] = (
```

Comment:
The reason is that we didn't have a `self.nuclides` or `self.elements` property. I just added them in, although there is no setter since `add_nuclide` and `add_element` methods are cleaner than directly allow a user to change the properties.